mmv1/third_party/terraform/services/secretmanager/resource_secret_manager_secret_test.go.tmpl

```diff
@@ -464,6 +464,31 @@ func TestAccSecretManagerSecret_updateBetweenTtlAndExpireTime(t *testing.T) {
 	})
 }
 
+func TestAccSecretManagerSecret_DeletionProtection(t *testing.T) {
```

Comment:
this test needs another set of steps to set deletion protection to false. As an update test, it should also include ConfigPlanChecks as described at https://googlecloudplatform.github.io/magic-modules/test/test/#add-an-update-test